x-pack/plugins/uptime/common/runtime_types/monitor/state.ts

```diff
@@ -105,7 +68,7 @@ export const MonitorSummaryType = t.intersection([
 
 export type MonitorSummary = t.TypeOf<typeof MonitorSummaryType>;
 
-export const MonitorSummaryResultType = t.intersection([
+export const MonitorSummariesResultType = t.intersection([
```

Comment:
This type is a duplicate of GetMonitorStatesResult, i believe we can remove that.